css/style.css

```diff
@@ -0,0 +1,682 @@
+  display: flex;
+  align-items: center;
+}
+
+.heading__logo {
+  margin-top: 4px;
+  margin-right: 45px;
+}
+
+.navigation__link {
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000;
+  margin-right: 30px;
+}
+
+.navigation__link:hover {
+  color: #b5b5b5;  
+}
+
+.navigation__link:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0,0,0,0.5);
+  -webkit-text-fill-color: rgba(0,0,0,0.1);
+}
+
+.heading__hamburger-wrapper {
+  display: none;
+}
+
+.search-form__input {
+  height: 27px;
+  width: 185px;
+  border-bottom: 1px solid #000;
+  background-color: #fff;
+  display: flex;
+  margin-right: 35px;
+}
+
+.search-form__text {
+  font-weight: 400;
+  border: none;
+  outline: none;
+  width: 316px;
+  vertical-align: middle;
+  color: #6c6c6c;
+  font-size: 14px;
+}
+
+.search-form__submit {
+  padding-top: 30px;
+  padding-right: 16px;
+  width: 14px;
+  height: 14px;
+  border: none;
+  background: url(../icons/search.png) no-repeat 50% 50%;
+}
+
+.heading__extras {
+  display: flex;
+  align-items: center;
+}
+
+.heading__signin {
+  text-decoration: none;
+  font-weight: 700;
+  color: #000;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  margin-right: 35px;
+}
+
+.heading__signin:hover {
+  color: #b5b5b5;
+}
+
+.heading__signin:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0,0,0,0.5);
+  -webkit-text-fill-color: rgba(0,0,0,0.1);
+}
+
+.heading__cart-wrapper {
+  display: flex;
+  align-items: center;
+  margin-right: 35px;
+}
+
+.heading__cart-link {
+  text-decoration: none;
+  font-weight: 700;
+  color: #000;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  margin-right: 4px;
+}
+
+.heading__cart-link:hover {
+  color: #b5b5b5;
+}
+
+.heading__cart-link:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0,0,0,0.5);
+  -webkit-text-fill-color: rgba(0,0,0,0.1);
+}
+
+.heading__cart-img {
+  height: 15px;
+}
+
+.heading__flag {
+  width: 20px;
+}
+
+/* Второй блок */
+
+.main__wrapper {
+  height: 280px;
+  background: url(../img/banner-bg.png);
+  margin-bottom: 52px;
+}
+
+.main__text-wrapper {
+  padding-top: 89px;
+  padding-left: 60px;
+  max-width: 195px;
+}
+
+.main__header {
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 29px;
+  text-transform: uppercase;
+  color: #fff;
+  padding-bottom: 12px;
+}
+
+.main__link {
+  position: relative;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.main__link::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 78px;
+  bottom: 2px;
+  background: url('../icons/arrow.png');
+  width: 45px;
+  height: 8px;
+}
+
+.main__link:hover {
+  color: #b5b5b5;
+}
+
+.main__link:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0,0,0,0.5);
+  -webkit-text-fill-color: rgba(255,255,255,.4);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5575166)